artiq/frontend/artiq_ddb_template.py

```diff
@@ -294,6 +294,18 @@ def process_mirny(self, rtio_offset, peripheral):
             name=mirny_name,
             refclk=peripheral["refclk"],
             clk_sel=clk_sel)
+        almazny = peripheral.get("almazny")
+        if almazny == True:
```

Comment:
Is Almazny on a Mirny simply enabled by adding ``almazny: true`` in the Mirny JSON description? This sounds fine to me.
But don't write ``if x == True:``, just write ``if x:``. And it would be cleaner to write ``peripheral.get("almazny", False)``.
Doesn't the JSON schema need an update?